engine/rotationmanager/setup.go

```diff
@@ -0,0 +1,54 @@
+package rotationmanager
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/riverqueue/river"
+	"github.com/target/goalert/engine/processinglock"
+	"github.com/target/goalert/event"
+	"github.com/target/goalert/schedule/rotation"
+)
+
+const (
+	QueueName         = "rotation-manager"
+	PriorityScheduled = 1
+	PriorityEvent     = 2
+	PriorityLFW       = 4
+)
+
+var _ processinglock.Setupable = &DB{}
```

Comment:
is this empty setup correct?

Reply:
Yes, but I'll add a comment and switch to the nil version.

https://go.dev/doc/effective_go#blank_implements